bundler/lib/bundler/man/bundle-fund.1.ronn

```diff
@@ -0,0 +1,37 @@
+bundle-fund(1) -- List information about gems seeking funding assistance
+============================================================
+
+## SYNOPSIS
+
+`bundle fund` [*OPTIONS*]
+
+## DESCRIPTION
+
+**bundle fund** lists information about gems seeking funding assistance.
+
+## OPTIONS
+
+* `-g`, `--group=GROUP`:
+  Fetch funding information for a specific group.
+
+* `--no-color`:
+  Disable colorization in the output.
+
+* `-r NUM`, `--retry=NUM`:
+  Specify the number of times to attempt network commands.
+
+* `-V`, `--verbose`, `--no-verbose`:
+  Enable verbose output mode.
```

Comment:
`--no-color`, `-r` and `-V` are global options for all commands that no command currently lists in its man page. So I'd remove them and stick to `--group`.